appshell/cefclient.cpp

```diff
@@ -95,7 +97,25 @@ void AppGetSettings(CefSettings& settings, CefRefPtr<CefCommandLine> command_lin
       command_line->GetSwitchValue(client::switches::kJavascriptFlags);
     
   // Enable dev tools
-  settings.remote_debugging_port = REMOTE_DEBUGGING_PORT;
+  CefString debugger_port = command_line->GetSwitchValue("remote-debugging-port");
+  if (!debugger_port.empty()) {
+    const long port = strtol(debugger_port.ToString().c_str(), NULL, 10);
+    if (errno == ERANGE) {
+        LOG(ERROR) << "Error while parsing remote-debugging-port arg: "<< debugger_port.ToString();
+        errno = 0;
+    }
+    static const long max_port_num = 65535;
```

Comment:
all of this should then be in the else case if errno is not ERANGE, since we know the port was not parsed properly and comparisons further may not give desired results.

Reply:
Done.